tests/Testcontainers.Tests/Fixtures/Containers/Unix/Modules/Databases/NoAuthMongoDbFixture.cs

```diff
@@ -0,0 +1,40 @@
+namespace DotNet.Testcontainers.Tests.Fixtures
+{
+  using System.Threading.Tasks;
+  using DotNet.Testcontainers.Builders;
+  using DotNet.Testcontainers.Configurations;
+  using DotNet.Testcontainers.Containers;
+  using MongoDB.Driver;
+
+  public sealed class NoAuthMongoDbFixture : DatabaseFixture<MongoDbTestcontainer, IMongoDatabase>
```

Comment:
FYI: You never use the fixture in a test. I'll fix it as soon as I merge the pull request.